Makefile

```diff
@@ -31,6 +31,10 @@ vet:
 lint: golangci-lint
 	$(GOLANGCI_LINT) run
 
+# todo(camilamacedo86): do not work
+# $ make generate
+  #/Users/camilamacedo/go/bin/controller-gen object:headerFile=./hack/boilerplate.go.txt paths="./..."
+  #open ./hack/boilerplate.go.txt: no such file or directory
 # Generate code
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
```

Comment:
not work.

Reply:
You are right. In Kb, wee use the controller-gen just to gen the testdata. 
We have no markers in the main.go as well. So, we do not need it.

Reply:
Done a pr to fix it.